src/components/ReportActionItem/ChronosOOOListActions.tsx

```diff
@@ -1,69 +1,68 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
 import Button from '@components/Button';
 import OfflineWithFeedback from '@components/OfflineWithFeedback';
 import Text from '@components/Text';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import DateUtils from '@libs/DateUtils';
-import reportActionPropTypes from '@pages/home/report/reportActionPropTypes';
 import useThemeStyles from '@styles/useThemeStyles';
 import * as Chronos from '@userActions/Chronos';
+import type {OriginalMessageChronosOOOList} from '@src/types/onyx/OriginalMessage';
+import type {ReportActionBase} from '@src/types/onyx/ReportAction';
 
-const propTypes = {
+type ChronosOOOListActionsProps = {
     /** The ID of the report */
-    reportID: PropTypes.string.isRequired,
+    reportID: string;
 
     /** All the data of the action */
-    action: PropTypes.shape(reportActionPropTypes).isRequired,
-
-    ...withLocalizePropTypes,
+    action: ReportActionBase & OriginalMessageChronosOOOList;
 };
 
-function ChronosOOOListActions(props) {
+function ChronosOOOListActions({reportID, action}: ChronosOOOListActionsProps) {
     const styles = useThemeStyles();
-    const events = lodashGet(props.action, 'originalMessage.events', []);
+
+    const {translate, preferredLocale} = useLocalize();
+
+    const events = action?.originalMessage?.events ?? [];
 
     if (!events.length) {
         return (
-            <View style={[styles.flexRow, styles.alignItemsCenter, styles.pt, styles.ml18]}>
+            <View style={[styles.flexRow, styles.alignItemsCenter, styles.ml18]}>
                 <Text>You haven&apos;t created any events</Text>
             </View>
         );
     }
 
     return (
-        <OfflineWithFeedback pendingAction={lodashGet(props.action, 'pendingAction', null)}>
-            <View style={[styles.chatItemMessage]}>
-                {_.map(events, (event) => {
-                    const start = DateUtils.getLocalDateFromDatetime(props.preferredLocale, lodashGet(event, 'start.date', ''));
-                    const end = DateUtils.getLocalDateFromDatetime(props.preferredLocale, lodashGet(event, 'end.date', ''));
+        <OfflineWithFeedback pendingAction={action?.pendingAction}>
```

Comment:
```suggestion
        <OfflineWithFeedback pendingAction={action.pendingAction}>
```

`action` can't be null/undefined here.

Reply:
removed